lib/src/core/sound_player.util.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:audioplayers/audioplayers.dart';
+
+final defaultClickSoundSource = AssetSource('packages/gaze_interactive/lib/assets/click.mp3');
+final player = AudioPlayer()..setSource(defaultClickSoundSource);
+
+enum SoundVolume {
```

Comment:
Ich würde hier die Enums rausziehen und in eigene Dateien machen. Hier sind ja mehr enums drin als wirklich eine Util Klasse